src/styles/stars.scss

```diff
@@ -0,0 +1,36 @@
+@import '../styles/variables';
+
+.card {
+  &__stars-reviews-wrapper {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__star {
+    display: flex;
+    width: $star-size;
+    height: $star-size;
+    margin-right: $small-margin;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__stars {
+    display: flex;
+  }
```

Comment:
Since &__stars is an element of &__card and does not have any additional styles, this block can be removed if it is not used for any specific reason. If it is used only for applying flex to direct children, consider combining it with the &__stars-reviews-wrapper rules if the HTML structure allows for it.